dataflowengineoss/src/main/scala/io/shiftleft/dataflowengineoss/passes/reachingdef/ReachingDefProblem.scala

```diff
@@ -150,8 +152,8 @@ class ReachingDefTransferFunction(flowGraph: ReachingDefFlowGraph) extends Trans
             .l
           mutable.BitSet(
             (retVal ++ args)
-              .filter(x => flowGraph.nodeToNumber.contains(x))
-              .map(x => Definition.fromNode(x.asInstanceOf[StoredNode], flowGraph.nodeToNumber)): _*
+              .filter(x => nodeToNumber.contains(x))
+              .map(x => Definition.fromNode(x.asInstanceOf[StoredNode], nodeToNumber)): _*
```

Comment:
filter+map == collect
A matter of taste, I guess. Same applies below. 
```suggestion
              .collect { case x: StoredNode if nodeToNumber.contains(x) =>
                Definition.fromNode(x, nodeToNumber)
              }: _*
```